### ISSUE TITLE
Stuck on Welcome screen and other issues

### ISSUE THREAD

**Issue by KesPCM** [zeffy/wufuc#120]
Hi Zeffy,

Thanks for this patch, but I have some issues with it.

After installing the patch and rebooting the PC for the first time, it got back to the login screen, I was able to type in my credentials, but then it got stuck at the Welcome screen with that little blue circle spinning.

I have hard powered off the PC and switched it on again, this time I was able to login and get into Windows. I then run Windows updates and was able to get them installed. Rebooted my PC, it came back, searched and installed more updates. Rebooted again. Came back fine one more time.

Then as I carried on using my PC I had to restart it at some point. And this time it ended up with that black screen and only cursor on it. Rebooted into safe mode with CMD, went into Regedit, but svchost.exe sub key is not listed under HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options, therefore nothing to delete in here.

Just rebooted my PC again and was able to login to Windows and use everything just fine. After a while had to reboot it one more time. This time I've got to the black screen and cursor only. Rebooted, got to type my credentials in, but hot stuck at Welcome message and the spinning wheel. Rebooted again, and now I;m back into my desktop.

So the whole thing is really random/intermittent. Not sure what exactly can I do to rectify this, but any help would be highly appreciated.

Cheers,
Kes

**Comment by zeffy** [comment]
What version of wufuc are you using? 0.7.1 should not be causing black screen errors, as far as I know only 0.8 could have caused it, but I pulled it from the downloads section a few weeks ago (readme has not been updated to reflect this). 

Does this keep happening if you uninstall wufuc?

What is your operating system, Windows 7 or 8.1, x86 or x64?

**Comment by KesPCM** [comment]
Hi Zeffy,

Thanks for your response. I have downloaded whichever version is currently available here on GitHub. It's 0.7.1.81. The PC is running Win7 x64.

I haven't tried uninstalling wufuc yet, as I wasn't really up for having to keep on re-installing it over and over again. It's been much better since yesterday, I was able to login to the PC without any issues ever since posting my firs message. But I'm just not sure if I can trust that the issues described in my original post won't keep on re-appearing from time to time. I don't want to reinstall wufuc now as the PC is working fine. Might try that if I get another black screen or stuck with the Welcome message.

Anyway. Let me know if you want any more specific details about my system/setup or any log files if that can help you developing the tool further.

Cheers,
Kes

**Comment by zeffy** [comment]
Hmm, I don't really know tbh. I'm kind of bewildered though, because I haven't seen this issue being caused by 0.7.1.81 before, only on the 0.8 version which was pulled, so if you are indeed using 0.7.1.81, that is concerning.

If you open regedit (not in safe mode) and navigate to the `Image File Execution Options` key, is there a `svchost.exe` subkey? If not, then you're definitely on 0.7.1.81. Also, can you open the Event Viewer and navigate to `Windows Logs > System` and see if there are many reports of services crashing? You don't have to copy all the entries here, just see if there are any reported around the times you experienced the black screen problem. That's all I can think of for now.

I'm currently working on a new version at the moment though, so I might have to put this on hold to see if the problem even still exists in the next version.

**Comment by KesPCM** [comment]
Hi Zeffy,

I've checked under Programs and Features and can confirm that I have 0.7.1.81. I have also checked for svchost.exe under Image File Execution Options (not in safe mode) and it's not there.

I've checked System logs and could see quite a few service crash reports around the time when I had black screen or was stuck at Welcome screen.

However I haven't had anymore of these issues yesterday and today, so it looks like my system just needed some time to "break the ice" with wufuc and now they are good friends. :) Hope it stays like that.
